packages/client/src/utils/index.ts

```diff
@@ -119,37 +122,50 @@ export async function buildFileTree(
 
 export async function getVercelIgnore(
   cwd: string | string[],
-  prebuilt?: boolean
+  prebuilt?: boolean,
+  rootDirectory?: string
 ): Promise<{ ig: Ignore; ignores: string[] }> {
-  const ignores: string[] = prebuilt
-    ? ['*', '!.output', '!.output/**']
-    : [
-        '.hg',
-        '.git',
-        '.gitmodules',
-        '.svn',
-        '.cache',
-        '.next',
-        '.now',
-        '.vercel',
-        '.npmignore',
-        '.dockerignore',
-        '.gitignore',
-        '.*.swp',
-        '.DS_Store',
-        '.wafpicke-*',
-        '.lock-wscript',
-        '.env.local',
-        '.env.*.local',
-        '.venv',
-        'npm-debug.log',
-        'config.gypi',
-        'node_modules',
-        '__pycache__',
-        'venv',
-        'CVS',
-        '.output',
-      ];
+  let ignores: string[] = [];
+
+  const outputDir = join(rootDirectory || '', '.output');
+
+  if (prebuilt) {
+    ignores.push('*');
+    const parts = outputDir.split('/');
+    parts.forEach((_, i) => {
+      const level = parts.slice(0, i + 1).join('/');
+      ignores.push(`!${level}`);
+    });
+    ignores.push(`!${outputDir}/**`);
+  } else {
+    ignores = [
+      '.hg',
+      '.git',
+      '.gitmodules',
+      '.svn',
+      '.cache',
+      '.next',
+      '.now',
+      '.vercel',
+      '.npmignore',
+      '.dockerignore',
+      '.gitignore',
+      '.*.swp',
+      '.DS_Store',
+      '.wafpicke-*',
+      '.lock-wscript',
+      '.env.local',
+      '.env.*.local',
+      '.venv',
+      'npm-debug.log',
+      'config.gypi',
+      'node_modules',
+      '__pycache__',
+      'venv',
+      'CVS',
+      `${outputDir}`,
```

Comment:
I think we still want to ignore all `.output` directories.

For example, if I have a structure that looks like this:
```
.
└── packages
    ├── project-a
    │   └── .output
    └── project-b
        └── .output
```

Then I don't think I want to have `project-a/.output` when I deploy `project-b`, right?

Reply:
Good catch! Fixed in c92a88f